sideboard/lib/sa/__init__.py

```diff
@@ -80,9 +80,9 @@ def process_bind_param(self, value, dialect):
             return str(value)
         else:
             if not isinstance(value, uuid.UUID):
-                return '%.32x' % uuid.UUID(value)
+                return uuid.UUID(value).hex
```

Comment:
This actually fails on python 3.5 and above:
```
Python 3.6.0 (default, Jan 23 2017, 11:13:53) 
[GCC 4.2.1 Compatible Apple LLVM 8.0.0 (clang-800.0.42.1)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import uuid
>>> '%.32x' % uuid.uuid4()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: %x format: an integer is required, not UUID
```

I ran into this error because I added python 3.5 to sideboard's test environments. It was a small enough change I didn't think it warranted a new issue and separate pull request.